src/tcpstream.c

```diff
@@ -34,13 +36,13 @@ int n_read(int fd, char *buffer, size_t total)
 	return total - left;
 }
 
-int n_write(int fd, const char *buffer, size_t total)
+int n_send(int fd, const char *buffer, size_t total)
 {
 	register ssize_t rtn;
 	register size_t left = total;
 
 	while (left > 0) {
-		rtn = write(fd, buffer, left);
+		rtn = send(fd, buffer, left, 0);
```

Comment:
Thanks for the code.

I believe send() is equivalent to write() if the flags is 0. 
Why we have to make this change? Thanks!

Reply:
Same as above:
I changed this to make the code easier to search: send/recv are more intuitive functions for a send/recv microbenchmark. On my test machine, there was an unexpected 1% performance improvement, too.